pkg/app/handlers/users/handler.go

```diff
@@ -135,19 +135,11 @@ func (u UsersResourceHandler) removeUserFromGroup(ctx context.Context, userID, g
 	return err
 }
 
-func (u UsersResourceHandler) setIdentity(ctx context.Context, userID, identity, kind string) error {
-	propsMap := make(map[string]interface{})
-	propsMap["kind"] = kind
-	props, err := structpb.NewStruct(propsMap)
-	if err != nil {
-		return err
-	}
-
-	_, err = u.dirClient.Writer.SetObject(ctx, &dsw.SetObjectRequest{
+func (u UsersResourceHandler) setIdentity(ctx context.Context, userID, identity string) error {
+	_, err := u.dirClient.Writer.SetObject(ctx, &dsw.SetObjectRequest{
 		Object: &dsc.Object{
-			Type:       "identity",
-			Id:         identity,
-			Properties: props,
```

Comment:
Don't we want to set some properties on the imported identity? At least in ds0 we have logic that expects the identity kind to be IDENTITY_KIND_PID when resolving OIDC JWTs.

Reply:
ah, I'll add it back